src/frontend/src/common/constants/envVariables.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  API_URL: process.env.API_URL,
+  BUILD_PATH: process.env.BUILD_PATH,
+  COMMIT_SHA: process.env.COMMIT_SHA,
+  NODE_ENV: process.env.NODE_ENV,
+};
```

Comment:
I think this would be more attractive to use if the object was simply ENV, e.g. `ENV.API_URL`

Reply:
Ooh this is a great idea! Thanks for the suggestion 🔥 Updated everything to use `ENV` 🚀 👏